src/dishka/integrations/aiohttp.py

```diff
@@ -93,10 +103,11 @@ def setup_dishka(
     app: Application,
     *,
     auto_inject: bool = False,
+    inject_decorator: InjectDecorator = inject,
```

Comment:
I do not like `inject_decorator` as a separate parameter here as it is only applicable if `auto_inject` is enabled.